images/awx-ee/execution-environment.yml

```diff
@@ -1,21 +1,58 @@
 ---
-version: 1
+version: 3
+
 dependencies:
+  ansible_core:
+    package_pip: ansible-core==2.13.6
+  ansible_runner:
+    package_pip: ansible-runner
   galaxy: requirements.yml
   python: requirements.txt
-  system: bindep.txt
+
+images:
+  base_image:
+    name: python:3.10
 
 additional_build_steps:
-  prepend:
+  # Source for those scripts are at https://github.com/ansible/ansible-builder/tree/devel/src/ansible_builder/_target_scripts
+  prepend_base: []
+  append_base: []
+
+  prepend_final:
     - LABEL org.opencontainers.image.authors="Digital Victoria"
     - LABEL org.opencontainers.image.description="Provides an AWX execution environment image optimised for use with SDP."
     - LABEL org.opencontainers.image.source="https://github.com/dpc-sdp/bay/blob/5.x/images/awx-ee/context/Dockerfile"
```

Comment:
Just a note to self: once this is merged, update the HEAD of #194 and move these assignments into the bake-file to keep them consistent.